Gemfile

```diff
@@ -50,6 +50,7 @@ end
 group :test do
   gem 'coveralls', require: false
   gem 'database_cleaner', '~> 1.7'
+  gem 'rails-controller-testing'
   gem 'rspec-activemodel-mocks', '~> 1.1.0'
   gem 'rspec-rails', '~> 3.8.0'
```

Comment:
bump  the followinjg gem versions `rspec-rails` to `3.9`, `database-cleaner` to `1.8` and `vcr` to `5.1.0`